ts/features/wallet/cobadge/component/__test__/CobadgeCard.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { render } from "@testing-library/react-native";
+import * as React from "react";
+import { ImageSourcePropType } from "react-native";
+import { Store } from "redux";
+import { Provider } from "react-redux";
+import configureMockStore from "redux-mock-store";
+// import { pot } from "italia-ts-commons";
+import { none, some } from "fp-ts/lib/Option";
+import * as hooks from "../../../onboarding/bancomat/screens/hooks/useImageResize";
+import CobadgeCard from "../CobadgeCard";
+import defaultCardIcon from "../../../../../../img/wallet/cards-icons/unknown.png";
+
+jest.mock("../../../onboarding/bancomat/screens/hooks/useImageResize");
+
+const aCaption = "****1234";
+const anAbiLogo = "http://127.0.0.1:3000/static_contents/logos/abi/03069.png";
+const aBrandLogo = defaultCardIcon;
+const anExpireMonth = "6";
+const anExpireYear = "2021";
+describe("BPayWalletPreview component", () => {
```

Comment:
```suggestion
describe("CoBadgeWalletPreview component", () => {
```

Reply:
Fixed in [0280969](https://github.com/pagopa/io-app/pull/2751/commits/028096919fd76eea38b2c405553e497075326f52)